src/Hyperion/Extensions/TypeEx.cs

```diff
@@ -130,11 +130,26 @@ private static Type GetTypeFromManifestName(Stream stream, DeserializerSession s
                     shortName = shortName.Replace("mscorlib,%core%", "System.Private.CoreLib,%core%");
                 }
 #endif
+                return LoadTypeByName(shortName);
+            });
+        }
 
-                var typename = ToQualifiedAssemblyName(shortName);
+        public static Type LoadTypeByName(string name)
+        {
+            try
```

Comment:
Initially I was going to just ignore assembly version, if specified.

But actually this may bring some issues in cases when there are multiple assembly versions available for loading (i.e. one available in GAC and another available locally), and older version of serializes are used on remote end, so it is passing assembly version.

So let's say if I am using old Akka.NET serializer on `node1` and I am updating Hyperion version on `node2`, and `node1` tells to use version `1.0.0.0` but I have `2.0.0.0` in GAC and `1.0.0.0` locally - I will get type loaded from `2.0.0.0`, and this may break something.  
While after updating serializer on remote end I will get this conflicts anyway (because assembly version will be missing and `2.0.0.0` will be loaded anyway) - this is not related to Hyperion, and I think we should keep backwards capability if possible.

The only drawback here is that throwing/catching exception may be expensive in terms of performance - but this should be fine, since exception will be thrown/catched only in cases when if was crashing before, and you can always improve performance by updating serializers to latest versions.